src/pylorax/ltmpl.py

```diff
@@ -146,13 +149,16 @@ class LoraxTemplateRunner(object):
 
     * Commands should raise exceptions for errors - don't use sys.exit()
     '''
-    def __init__(self, inroot, outroot, yum=None, fatalerrors=True,
-                                        templatedir=None, defaults={}):
+    def __init__(self, inroot, outroot, _yum=None, fatalerrors=True,
```

Comment:
Use yum_obj instead.

_var naming is normally used for unused variables, if there is a name collision it should be given a new descriptive name.